src/nimble_kokkos_model_data.cc

```diff
@@ -431,14 +431,33 @@ void ModelData::UpdateStates(const nimble::DataManager &data_manager)
 }
 
 
-Viewify ModelData::GetScalarNodeData(const std::string& label)
+nimble::Viewify<1> ModelData::GetScalarNodeData(const std::string& label)
 {
   auto field_id = GetFieldId(label);
   if (field_id < 0) {
     std::string code = " Field " + label + " Not Allocated ";
     throw std::runtime_error(code);
   }
-  return {GetHostScalarNodeData(field_id).data(), 1};
+  auto field_view = GetHostScalarNodeData(field_id);
+  auto field_size = static_cast<int>(field_view.extent(0));
+  auto field_stride = static_cast<int>(field_view.extent(0));
```

Comment:
"extent" is assigned to "stride" but below I see a view.stride?

Reply:
You are right. Good catch